src/app/FinalLevelExam/FinalLevelExam-detail/FinalLevelExam-detail.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import {FinalLevelExam} from '../FinalLevelExam';
+import {ActivatedRoute} from '@angular/router';
+import {AuthenticationBasicService} from '../../login-basic/authentication-basic.service';
+import {User} from '../../login-basic/user';
+import {FinalLevelExamService} from '../FinalLevelExam.service';
+
+@Component({
+  selector: 'app-FinalLevelExam-detail',
+  templateUrl: './FinalLevelExam-detail.component.html',
+})
+  export class FinalLevelExamDetailComponent implements OnInit {
+  public finalLevelExam: FinalLevelExam = new FinalLevelExam();
+
+  constructor(private route: ActivatedRoute,
+              private FinalLevelExamService: FinalLevelExamService,
+              private authenticationService: AuthenticationBasicService) {
+  }
+
+  ngOnInit() {
+    const id = this.route.snapshot.paramMap.get('id');
+    this.FinalLevelExamService.get(id).subscribe(
+      finalLevelExam => {
+        this.finalLevelExam = finalLevelExam;
```

Comment:
The questions for the final exam can be retrieved following the corresponding _link. This is facilitated by the ngx_hal_client dependency as I showed in the last recording. Something like:

```
this.finalLevelExam.getRelationArray(Question, 'questions').subscribe(questions => this.finalLevelExam.questions = questions))
```